src/machines/java/com/enderio/machines/common/block/MachineBlock.java

```diff
@@ -94,6 +94,8 @@ public InteractionResult use(BlockState state, Level level, BlockPos pos, Player
         //pass on the use command to corresponding block entity.
         InteractionResult result = ((MachineBlockEntity)entity).onBlockEntityUsed(state, level, pos, player, hand,hit);
         if (result != InteractionResult.CONSUME) {
+            if (level.getBlockEntity(pos) instanceof MachineBlockEntity machine && !machine.canOpenMenu())
```

Comment:
What is this change for?

Reply:
even though MenuProvider are contractually allowed to provide nullable menus, they throw a npe in the packet somewhere, which is logged, not really nice this solution, but I didn't want nullpointers from us logged

Reply:
Aha makes sense, thanks :)